MarvelData/Data/Services/APIServices.swift

```diff
@@ -0,0 +1,116 @@
+//
+//  APIServices.swift
+//  MarvelData
+//
+//  Created by Swapnali Patil on 20/04/22.
+//
+
+import Foundation
+import Alamofire
+
+typealias ResponseCompletion<T: Decodable> = (T?, _ error: Error?) -> Void
```

Comment:
Why not use standard swift `Result` type? 
something like 
```suggestion
typealias ResponseCompletion<T: Decodable> = (Result<T, Error>) -> Void
```